res/values-ro/strings.xml

```diff
@@ -128,7 +128,7 @@
     <string name="Friend_Request_Sent">"Cerere de prietenie trimisă"</string>
     <string name="ERROR">"EROARE"</string>
     <string name="CONTINUE">"CONTINUĂ"</string>
-    <string name="Global_" formatted="false">"Global%"</string>
+    <string name="Global_" formatted="false">"Global formatata fals"</string>
```

Comment:
You shouldn't translate and provide the `formatted=false`. "Global" seems better here.